configure.ac

```
@@ -297,11 +297,17 @@ AC_ARG_ENABLE([debug-runtime],
   [AS_HELP_STRING([--disable-debug-runtime],
     [do not build runtime with debugging support])])
 
-AC_ARG_ENABLE([debugger],
+AC_ARG_ENABLE([ocamldebug],
   [AS_HELP_STRING([--enable-debugger],
```

Spotted by @shindere:
```suggestion
  [AS_HELP_STRING([--enable-ocamldebug],
```